.github/workflows/trivy.yml

```diff
@@ -0,0 +1,53 @@
+name: Trivy
+on: workflow_dispatch
+
+jobs:
+  trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Build all projects without running tests
+        run: ./gradlew --build-cache build -x test -x spotlessCheck
+
+      - name: Construct docker image name and tag
+        id: image-name
```

Comment:
Now that this is a completely separate operation from the PR workflow, I think it would be better to use a fixed name here, so it's clear that when an error is found, it's in the locally built image, and not in the image in artifcatory / gcr.

Reply:
I added a fixed name.